sys/include/pm_layered.h

```diff
@@ -52,8 +53,7 @@ extern "C" {
 /**
  * @brief Power Management mode blocker typedef
  */
-typedef union {
-    uint32_t val_u32;                   /**< power mode blockers u32 */
+typedef struct {
     uint8_t val_u8[PM_NUM_MODES];       /**< power mode blockers u8 */
```

Comment:
Hmm, I wonder if that can be considered as internal API and we could rename this to something that is easier to understand. In absence of `val_u32`, the `val_u8` now makes little sense anymore.

Reply:
Ok, I renamed it to `blockers`